### DISCUSSION TITLE
Dreambooth: dumping ckpt at set intervals?

### DISCUSSION THREAD

**Discussion by giteeeeee** [TheLastBen/fast-stable-diffusion#208] [Q&A] [answered]
For dreambooth, is there a function to generate ckpt files at user-defined intervals? Something like output1:___ steps, output2:___ steps, output3:___ steps?

edit:
https://github.com/TheLastBen/fast-stable-diffusion/issues/206

I see it's been working on. I'll keep an eye on this post.

**Comment by TheLastBen** [comment, marked as answer]
Yes, I will add it shortly

**Reply by giteeeeee** [reply]
thanks :)

**Comment by TheLastBen** [comment]
Feature added https://github.com/TheLastBen/diffusers/commit/0b77c4e56bf377d424c7ed87d6cb1804469b0217
https://github.com/TheLastBen/fast-stable-diffusion/commit/a8de378e12df1a3b155f6750f02072b7d75290c6
